.github/workflows/spanner-integration-tests-against-emulator.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '2.6'
+          ruby-version: '3.0'
```

Comment:
Any reason not to just go straight to 3.2?

Reply:
Nothing specifically, was trying to resolve error with 2.6. Have now moved to 3.2 👍